lib/elixir/lib/system.ex

```diff
@@ -754,20 +755,23 @@ defmodule System do
     :erlang.unique_integer(modifiers)
   end
 
-  defp convert_time_unit(:native),       do: :native
-  defp convert_time_unit(:seconds),      do: :seconds
-  defp convert_time_unit(:milliseconds), do: :milli_seconds
-  defp convert_time_unit(:microseconds), do: :micro_seconds
-  defp convert_time_unit(:nanoseconds),  do: :nano_seconds
+  defp normalize_time_unit(:native),                                 do: :native
```

Comment:
I think a whitespace longer than function definition means we need to move `do:` on the next line. :bowtie: